src/utils/translateFromContent.js

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from 'react-i18next';
+import i18n from 'i18next';
+
+// Translates content based on the current language setting
+// Handles translation of content objects that contain language-specific text
+export function translateFromContent(content) {
+	const { t } = useTranslation();
+	const currentLang = i18n?.language || 'c';
+	const errorMessage = t("General|Content cannot be translated - invalid format");
```

Comment:
`i18next.t('General|Content cannot be translated - invalid format');`

This is how you can do translation without hooks